### ISSUE TITLE
Ruby version incorrectly reported with -V

### ISSUE THREAD

**Issue by rosswilson** [bbatsov/rubocop#1564]
I'm running ruby 2.2.0:

```
ruby --version
```

```
ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin13]
```

But when I run ```rubocop -V``` it prints:

```0.26.1 (using Parser 2.2.0.pre.4, running on ruby 2.1.3 x86_64-darwin12.0)```

**Comment by jonas054** [comment]
What about
```
ruby -S rubocop -V
```
What do you get?

**Comment by itsmrwave** [comment]
I have v2.1.5 set globally and v2.1.2 set in my app ... I can verify this using `ruby--version`. Oh ... and I'm using `rbenv`. When I run `rubocop -V` I get ...

```
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.5-compliant syntax, but you are running 2.1.2.
0.27.1 (using Parser 2.2.0.pre.8, running on ruby 2.1.2 x86_64-darwin14.0)
```

Seems Rubocop for some reason uses the ruby version set globally instead of what's set locally.

@jonas054 Running `ruby -S rubocop -V` gives me the same warning as above.

Ps: I have a `.ruby-version` in my app root containing the Ruby version of within that context i.e. 2.1.2 ... I'm not sure if this is a standard or just a feature of `rbenv`.

**Comment by jonas054** [comment]
My guess is that this is more of an `rbenv` issue than a `rubocop` issue.

@rosswilson Were you also using `rbenv`?

@itsmrwave Can you try with other gems that have a binary? Is it possible to see which ruby version they use?

**Comment by itsmrwave** [comment]
@jonas054 Do you have any in mind? Something that's commonly used. Few gems offer a verbose `-v` option. Tried to think of `brakeman`, `rspec` i.e. stuff that I call on the command line but all just output their version.

That said ... I've had no issue with `rbenv` setting the ruby version in the context within what's configured ... so not sure it's `rbenv`. Even running `ruby --version` within the same folder gives me the correct Ruby version.

Also I was trying to look into the code to see how Rubocop determines the version. Any pointers as to the specific part? I thought it checks for `RUBY_VERSION`. When I run `echo $RUBY_VERSION` in the folder I get blank output.

**Comment by bbatsov** [comment]
> I thought it checks for RUBY_VERSION. When I run echo $RUBY_VERSION in the folder I get blank output.

`RUBY_VERSION` is a Ruby constant, not an env variable. And yes - that's exactly what RuboCop checks, so I cannot imagine it being wrong. This problem is likely caused by rbenv.

**Comment by rosswilson** [comment]
@jonas054 I am using rbenv, yes. When I'm home I'll do some more digging into this issue.

**Comment by itsmrwave** [comment]
@bbatsov Ah, silly mistake :-) Thanks though.

**Comment by jonas054** [comment]
@itsmrwave Sorry, I don't know of any other gems that print the Ruby version. You might be able to hack one of them to do it. Just change the source code where the gem is installed.

**Comment by bbatsov** [comment]
Pretty sure this is not actually a RuboCop bug, so I'll close it.
